timer/runtime/worker.go

```diff
@@ -68,6 +70,13 @@ type hookWorker struct {
 	ch        chan *triggerEventRequest
 	logger    *zap.Logger
 	nowFunc   func() time.Time
+	// metrics for worker
+	triggerRequestCounter       prometheus.Counter
+	onPreSchedEventCounter      prometheus.Counter
```

Comment:
We can create metrics package to support metrics for timer. such as ```executor/metrics```.

Reply:
done